app/controllers/projects_controller.rb

```diff
@@ -42,6 +43,15 @@ def index
         @projects = current_power.projects
       end
       format.json do
+        only_updatable = ActiveModel::Type::Boolean.new.cast(params[:onlyUpdatable])
+
+        # TODO(mark): Reconcile this with the part below.
+        # These returned projects contain different fields than the ones below.
+        if only_updatable
+          render json: current_power.updatable_projects
+          return
```

Comment:
Should we apply the "Do One Thing and Do It Well" principle to these controller actions? Rather than having the early return here, it might make sense to have a separate index action for uploads